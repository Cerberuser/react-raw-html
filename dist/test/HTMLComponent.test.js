"use strict";
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
    result["default"] = mod;
    return result;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const React = __importStar(require("react"));
const react_test_renderer_1 = __importDefault(require("react-test-renderer"));
const src_1 = require("../src");
test("Raw HTML renders correctly", () => {
    const component = react_test_renderer_1.default.create(React.createElement(src_1.HTMLComponent, { rawHTML: "<span>Text</span>" }));
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=HTMLComponent.test.js.map