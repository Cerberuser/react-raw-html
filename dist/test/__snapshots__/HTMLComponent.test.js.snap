// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Raw HTML renders correctly 1`] = `
Array [
  <SPAN />,
  "Text",
  "Unknown node",
]
`;
